sql/dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mysql
+
+ENV MYSQL_DATABASE=liveqa \
```

Comment:
Does this Env-variable create the database `liveqa`? There is no need for IMHO, bacause it's created in `liveqa.sql`